### DISCUSSION TITLE
Problem with a grid I made myself (without gridgen)

### DISCUSSION THREAD

**Discussion by fed1942** [NOAA-EMC/WW3#1195] [Q&A] [answered]
I did a WaveWatch simulation using a regular 25x13 grid (1.25x1.25 min grid) and got good results. However, with another grid I made that covers the same area, 51x26 (0.6x0.6 min), I obtained extremely high Hs values (>10 m).

I used the multi grid version of WW3 with the following grids:
Global grid > South Atlantic grid > Buenos Aires grid (8x8 km) > Local grid (0.6x0.6 min)

I attach files for the local grid.
[ww3_grid_51x26.txt](https://github.com/NOAA-EMC/WW3/files/14406615/ww3_grid_51x26.txt)
[Archivo BOT_51x26.txt](https://github.com/NOAA-EMC/WW3/files/14406652/Archivo.BOT_51x26.txt)
[Archivo MASK_51x26.txt](https://github.com/NOAA-EMC/WW3/files/14406653/Archivo.MASK_51x26.txt)

Any ideas on what could be happening?

**Comment by ukmo-ccbunney** [comment, marked as answer]
@fed1942 Did you change your timesteps when you generated your higher resolution 0.6x0.6 km grid?
You model could be going unstable.

**Reply by fed1942** [reply]
No, i didnt change the timesteps. I forgot about that, i am a beginner in the subject of numerical models.

Any advice to set the timesteps?

Thanks for the reply.

**Reply by CarstenHansen** [reply]
There is a detailed prescription in the template namelist file WW3/model/nml/ww3_grid.nml (an alternative to ww3_grid.inp) starting with the line "! Define the timesteps parameterization via TIMESTEPS_NML namelist". The time steps are determined from the lowest frequency FREQ1 = 0.04118 Hz and the shortest grid cell size DXY ~ 867 m. This will result  in a maximum CFL time step DTXY on the order of 30 seconds, much shorter that your value of 150 s.

**Reply by ukmo-ccbunney** [reply]
To complement @CarstenHansen 's reply, see also Appendix A.1 ("_Setting model time steps"_) in the [WW3 Manual.](https://github.com/NOAA-EMC/WW3/wiki/Tech-Docs)
